bindgen/clang.rs

```diff
@@ -707,11 +708,12 @@ impl Cursor {
         }
 
         unsafe {
-            let w = clang_getFieldDeclBitWidth(self.x);
-            if w == -1 {
-                None
+            let n: c_int = clang_getFieldDeclBitWidth(self.x);
+            if n >= 0 {
+                Some((n as c_uint).try_into().unwrap())
             } else {
-                Some(w as u32)
+                debug_assert_eq!(n, -1);
+                None
             }
         }
```

Comment:
This is kind of confusing with both the `as` cast and `try_into`. This seems more readable:

```rust
let ret = c_uint::try_from(n).ok();
if ret.is_none() {
    debug_assert_eq(n, -1, "-1 is the only expected error");
}
ret
```